docker-run

```diff
@@ -14,14 +14,31 @@ if [ ! -f "$RESIN_IMAGE_FLASHER" ]; then
 	exit 1
 fi
 
+# join arguments to pass to the container...
+shift
+args=()
+while [[ $# -gt 1 ]]; do
+	args+=("$1 $2")
+	shift
+	shift
+done
+flasher_args="${args[*]}"
+
 # Get the absolute script location
 pushd "$(dirname "$0")" > /dev/null 2>&1
 SCRIPTPATH="$(pwd)"
 popd > /dev/null 2>&1
 
+echo "Building tool, please wait..."
+extraction_container="$(docker build -q -f Dockerfile.amd64 .)"
+
+resin_image_name="$(basename ${RESIN_IMAGE_FLASHER})"
+export USER_ID="$(id -u $USER)"
```

Comment:
Not sure the export is needed as USER_ID is just just below on this same script.

Reply:
I'll investigate @alexgg and remove if not needed 👍